Task8/README.md

```diff
@@ -0,0 +1,158 @@
+# ZenML pipeline downloading data from BQ in one step and running a zenML pipeline in Vertex AI pipelines
+
+## Notes
+In the folder you will find two python scripts: `pipeline.py` and `run.py`, `pipeline.py` showcases how to download data from BQ in zenML pipeline and 
+`run.py` is used to run another zenML pipeline (mnist) remotely in `Vertex AI` pipelines. The reasons for that is, that so far you can run only pipelines
+that contain python modules installable with `zenml integration install <package>` (list of packages is [here](https://docs.zenml.io/features/integrations)).
+And since to download data from `BigQuery` I used the recommended `pandas_gbq` (not included in zenML integrations) it fails when submitting the job to 
+`Vertex AI` (because zenML defined an image where they install only their integrated packages).
+
+Correction:
+
+That is true by default, but when you are registering a new step-operator you can define your custom base image `--base_image=<CUSTOM_BASE_IMAGE>` 
```

Comment:
What if you used the GCP default python package for BQ interaction google-cloud-bigquery. Is that included and would that avoid custom images?

Reply:
I tried it with `from google.cloud import bigquery` and the pipeline runs. So this would avoid custom images.